hugo-websites/deploy.sh

```diff
@@ -27,17 +27,17 @@ buildEnv() {
   environment="$(echo "${deployment}" | jq -r '.metadata.labels.environment')"
   image="$(echo "${deployment}" | jq -r '.spec.template.spec.containers[] | select(.name == "nginx") | .image')"
 
-  if [[ "${environment}" == "production" ]]; then 
+  if [[ "${environment}" == "production" ]]; then
     BASE_NGINX_IMAGE_TAG="stable-alpine"
   else
     BASE_NGINX_IMAGE_TAG="stable-alpine-for-staging"
   fi
 
-  docker build --pull --build-arg NGINX_IMAGE_TAG="${BASE_NGINX_IMAGE_TAG}" -t "${image}" -f "${DOCKERFILE}" "${CONTEXT}"
+  docker build --pull --build-arg NGINX_IMAGE_TAG="${BASE_NGINX_IMAGE_TAG}" -t "${image}" -f "${DOCKERFILE}" "${CONTEXT}" --no-cache
```

Comment:
I think context must be the last arg, so please move `--no-cache` after `--pull`

Reply:
I initially had this earlier in the command and the build failed. However, I might have inserted the option after --build-arg. 

What I have here does work but not clear to me if the --no-cache option was used. I will move it based on your recommendation.